bin/sapconf

```diff
@@ -218,7 +218,7 @@ case "$1" in
       exit 1
     fi
     profile=${profile:-sapconf-bobj}
```

Comment:
we need to change line 216 to check only for 'sapconf' profiles
if [[ $profile != "" && $profile == sapconf-* ]]; then
because only then a 'stop' action makes sense.

and we need to change the profile setting to an absolute setting
profile=sapconf-bobj
because the use of 'sapconf ' is exactly to set a defined 'sapconf' profile.